documentation/manual/src/main/asciidoc/en-US/modules/infinispan.asciidoc

```diff
@@ -74,7 +74,7 @@ which is a good starting point for new users - you don't have to set any propert
 
 .Hibernate OGM properties for Infinispan
 `hibernate.ogm.datastore.provider`::
-Set it to `infinispan` to use Infinispan as the datastore provider.
+Set it to `infinispan_embedded` to use Infinispan as the datastore provider.
```

Comment:
I'll change this instance, but I wouldn't go further at this stage. With the introduction of the remote dialect we'll likely need to refactor the whole "Infinispan" chapter to have a common introduction and then a split in subsections.